auth/comparePassword.js

```diff
@@ -0,0 +1,21 @@
+const bcrypt = require( 'bcrypt' )
+
+const findUser = ( User, email, password ) => {
+  return User.findOne({ where: { email, email_verified: true }})
+    .then( user =>
+      new Promise( (resolve, reject) => {
+        if( ! user ) {
+          reject({})
+        }
+        bcrypt.compare( password, user.password, (error, result ) => {
```

Comment:
Please add a new line above this line; otherwise it's hard to see where the if block ends and the processing logic begins.